lighthouse-extension/app/src/popup.js

```diff
@@ -141,7 +141,20 @@ document.addEventListener('DOMContentLoaded', _ => {
           ' Close the other tabs to use lighthouse.';
       }
 
+      const reportErrorEl = document.createElement('a');
+      reportErrorEl.className = 'button button--report-error';
+
+      const base = 'https://github.com/googlechrome/lighthouse/issues/new?';
+      const title = 'title=Lighthouse%20Extension%20Error';
+      const body = '&body=Error:%20' + message;
```

Comment:
the two cases handled by `if` statements above (devtools open and multiple tabs) should not trigger the report error button, since they're local things the user needs to fix